### DISCUSSION TITLE
What happened to the demo category in the firmware builder?

### DISCUSSION THREAD

**Discussion by Kimm0th** [joeycastillo/Sensor-Watch#391] [Q&A] [answered]
A few months ago I used the online tool to make myself a firmware, in which I tried a variety of faces. Having decided which ones I wanted to use going forward, now I find I can't include the voltage face because the demo category seems to have vanished from the page. 

Am I missing something? If I have to do it on my own machine instead of the online builder, is there a better how-to than the overly brief instructions on the official page? About all I was able to glean from that is it needs to be done on a Linux box... I have an RPi 400 and a Mac running Linux, but I don't know how to drive it. 

Any halp for a noob like me somewhere? I'm flat-out finding my way around GitHub.

**Comment by wryun** [comment, marked as answer]
Sorry about this - I left off the 'demo' category when I added categories to the builder last time I updated it. Should be there now.

By the way, if you have issues with the builder, you could raise them at https://github.com/wryun/sensor-watch-builder/

---

If you want to try building a firmware on your own machine, let us know where you get stuck with:

https://www.sensorwatch.net/docs/movement/building/

There are instructions for Linux and Mac. Sometimes it's hard when you already know things to see where the gaps are!

**Reply by shedsynctronics** [reply]
Hi, I have some feedback on trying to compile hope it's ok to post it here.....

I have been trying to compile the sensor watch but with no success so far :-/
Not sure I've installed everything correctly... which version of python is required (Win32, Win64 or ARM64)?
Also I did the command prompts to set the paths but nothing seemed to happen when i did.
And...., what exactly does "From within the [repo]" mean in the compile instructions... i took it to mean i had to download the repo to my local machine and open a command prompt at the repo location. 
I used Github desktop to clone it on my local PC and executed the commands from the repo location but got the following result.

C:\Users\cmaso\Documents\GitHub\Sensor-Watch\movement\make>make COLOR=RED
'make' is not recognized as an internal or external command,
operable program or batch file.

Hope someone can give me some pointers.
Thanks,
James

**Reply by wryun** [reply]
Hi James,

Discord is probably your best bet, or start a new discussion (this one is about the onlline firmware builder, not building yourself - https://sensor-watch-builder.fly.dev/).

I can't help much with Windows, but if you're not getting 'make' to do anything, this is a sign you haven't correctly installed the toolchain and/or set your PATH to use the toolchain.

**Comment by Yometalc** [comment]
Inside the repo means from whatever directory you cloned the GIT repo into afaik
